src/OpenIddict/OpenIddictOptions.cs

```diff
@@ -79,5 +79,11 @@ public class OpenIddictOptions {
         /// Setting this option to false in production is strongly encouraged to mitigate man-in-the-middle attacks.
         /// </summary>
         public bool AllowInsecureHttp { get; set; }
+
+        /// <summary>
+        /// Set to True to allow you to use your own views/styles/scripts in your server.
+        /// When using custom views you MUST provide cshtml views for: Authorize (consent), Logout, and SignIn
+        /// </summary>
+        public bool UseCustomViews { get; set; }
```

Comment:
Not sure how I feel about a single option that control both the assets and the views.